apps/morph_check

```diff
@@ -193,7 +193,7 @@ if __name__ == '__main__':
     yaml_path = args.config
     SEPARATOR = '=' * 40
 
-    config = {'neuron_checks': [], 'options': []}
+    config = {'neuron_checks': [], 'options': {}}
```

Comment:
Is this going to work correctly if you provide a yaml file? I'll check if it works.